### PR TITLE
Cristian development

### PR DESCRIPTION
**Pull request by cristianUrbina**

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -8,12 +8,106 @@
...
+          a computer scientist was a new oportunity to me to continue using and developing
+          skills that Mathematical Olympiads gave me.
+      </p>
+
+      <p>
+          I've been enroled in competitive programming for almost a year.
+          The last topic I've been working on is dynamic programming.
+      </p>
+      <ol>
+          <li>
```
Comment (KevinWorkman) [inline portfolio/src/main/webapp/index.html:49]: Google's HTML style guide recommends indenting by 2 spaces: https://google.github.io/styleguide/htmlcssguide.html#Indentation

I'm not going to be super strict about this kind of style guide rule though, so it's up to you what style you choose.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -17,7 +17,7 @@
  */
 function addRandomGreeting() {
   const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+      ['I love to eat bread', 'I like gossip, specially when it does not involve me','Architects is my favorite band', '\"I\'m ready\"-Sponge Bob'];
```
Comment (KevinWorkman) [inline portfolio/src/main/webapp/script.js:20]: I think you can probably get away without escaping the double quotes here. I think JavaScript is smart enough to recognize when one type of quotation punctuation is inside another. So you could make it this:

`'"I\'m ready" - Sponge Bob'`

**Review by KevinWorkman — APPROVED** [review]
Couple minor comments, but overall looks great to me!

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (cristianUrbina) [reply portfolio/src/main/webapp/index.html:49]: Oh, sure I will change that. Do you know a way to change it quickly?

**Pull request merged**